source/documentation/tools/jupyterlab/package-management.md

```diff
@@ -1,5 +1,89 @@
 # Python package management
 
+## Poetry
+We advise the use of [poetry](https://python-poetry.org/docs/) as a library.
+
+### <b>why use poetry</b> 
+poetry stops conflicts between libraries and their dependencies.
+It makes sure that libraries play well together.
+If virtualenv is used without poetry, you may start experiencing library version conflict and start getting warnings.
+
+### <b>setup steps:-</b>
+- after creating your virtualenv and source it
```

Comment:
I'm not sure most people would know what `source it` means.  It might be best to link to a virtualenv section and talk about activating it instead?